### PR TITLE
Plugins to view the buoy on RViz

### PR DESCRIPTION
**Pull request by quarkytale**
Adding [sdformat_urdf](https://github.com/ros/sdformat_urdf) to the project

### CODE REVIEW

**Comment by quarkytale** [comment]
Installing from source might not be required if we just end up using the binaries

**Comment by chapulina** [comment]
It turns out that we should actually be compiling `sdformat_urdf` from source instead of using the debs. The debs are being compiled against Citadel 😱 , and we need to use then with Fortress. I opened this PR upstream to add Fortress support from source:

* https://github.com/ros/sdformat_urdf/pull/8

Once that's merged, we should be able to use the `galactic` branch here, as this PR is suggesting. We'll just need to also set `GAZEBO_VERSION=fortress` next to `IGNITION_VERSION` in these places:

https://github.com/osrf/buoy_entrypoint/blob/f8cf6f312c4c5cf64d23ed73a149bb23ec1403ed/docker/buoy/Dockerfile#L11

https://github.com/osrf/buoy_entrypoint/blob/f8cf6f312c4c5cf64d23ed73a149bb23ec1403ed/README.md#L50

**Review by chapulina — CHANGES_REQUESTED** [review]
LGTM after `GZ_VERSION` is added to the places suggested above.

* We also need https://github.com/ros/sdformat_urdf/pull/8 or equivalent merged upstream.

**Review by chapulina — APPROVED** [review]

**Pull request merged**
